example/broker.cpp

```diff
@@ -16,18 +16,44 @@
 #include <fstream>
 
 #if defined(MQTT_USE_TLS)
-boost::asio::ssl::context init_ctx(boost::program_options::variables_map const& vm)
+constexpr std::size_t certificate_reload_interval_seconds = 3600;
+
+template<typename ServerPtr>
+void reload_ctx(boost::asio::io_context &ioc, ServerPtr server, boost::program_options::variables_map const& vm, std::string const &name)
```

Comment:
Why name is needed?

Reply:
For the info log message

Reply:
WSS and TLS use a different cert files?

Reply:
No, they have a different context. So they don't share a reference to the ssl context, but the context is moved into the Server class. So if a new file is loaded, it needs to be loaded into two different ssl context: WSS and TLS

Reply:
That is also why this is needed:

https://github.com/redboltz/mqtt_cpp/pull/781/commits/e9fa33cc4b2f4d3c9ed82c338ee1c1ddc0dc62cc#diff-555c49abb1b633967427e5b3374722f21d0dc4898b352d55a715168a1e761b00R62-R77

Which i don't like that much

Reply:
I understand why name is needed.

Reply:
name is `std::string const&`, and you pass "WSS". Implicit temporary `std::string` is created.
It is quite misleading.
Use `char const*` of `std::string`.